package.json

```diff
@@ -1,24 +1,31 @@
 {
```

Comment:
So I think it's a real problem, and I think the fix is to modify `coder.com/site/package.json` to add these dependencies, or modify that file to also install the dependencies in `docs` when the top-level build runs.

Here's what I did to repro the problem, which is obscured if you run `yarn` inside the `docs` submodule, because then it seems to use its local node_modules?

I recorded a script

```
Script started on 2021-03-29 20:09:35+00:00 [TERM="xterm-256color" TTY="/dev/pts/0" COLUMNS="274" LINES="55"]
]0;coder@jawnsy-m:~/projects/m/product/coder.com/site[?2004h[coder@jawnsy-m site]$ ls[K[Kpwd
[?2004l
/home/coder/projects/m/product/coder.com/site
]0;coder@jawnsy-m:~/projects/m/product/coder.com/site[?2004h[coder@jawnsy-m site]$ yarn
[?2004l
[2K[1G[1myarn install v1.22.10[22m
[2K[1G[2m[1/4][22m Resolving packages...
[1G⠁ [0K[2K[1G[2K[1G[2m[2/4][22m Fetching packages...
[2K[1G[1G[-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 0/872[1G[###############################----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 103/872[1G[############################################################-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 200/872[1G[###########################################################################################----------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 302/872[1G[########################################################################################################################-----------------------------------------------------------------------------------------------------------------------------------------------] 397/872[1G[########################################################################################################################################################---------------------------------------------------------------------------------------------------------------] 504/872[1G[##########################################################################################################################################################################################-----------------------------------------------------------------------------] 616/872[1G[############################################################################################################################################################################################################################-------------------------------------------] 728/872[1G[####################################################################################################################################################################################################################################################-------------------] 809/872[2K[1G[34minfo[39m fsevents@2.3.2: The platform "linux" is incompatible with this module.
[2K[1G[34minfo[39m "fsevents@2.3.2" is an optional dependency and failed compatibility check. Excluding it from installation.
[2K[1G[34minfo[39m fsevents@1.2.13: The platform "linux" is incompatible with this module.
[2K[1G[34minfo[39m "fsevents@1.2.13" is an optional dependency and failed compatibility check. Excluding it from installation.
[1G[#######################################################################################################################################################################################################################################################################] 872/872[2K[1G[2m[3/4][22m Linking dependencies...
[2K[1G[33mwarning[39m " > @material-ui/icons@4.11.2" has unmet peer dependency "@material-ui/core@^4.0.0".
[2K[1G[33mwarning[39m " > slick-carousel@1.8.1" has unmet peer dependency "jquery@>=1.8.0".
[2K[1G[2K[1G[1G[-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 0/934[2K[1G[2K[1G[1G[-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 0/40139[1G[##-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 380/40139[1G[#####----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 706/40139[1G[#######-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 1024/40139[1G[#########-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 1408/40139[1G[############--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 1789/40139[1G[##############------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 2170/40139[1G[################----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 2432/40139[1G[##################--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 2775/40139[1G[###################-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 2943/40139[1G[####################------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 3072/40139[1G[#####################-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 3273/40139[1G[######################----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 3453/40139[1G[########################--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 3648/40139[1G[##########################------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 3962/40139[1G[############################----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 4267/40139[1G[##############################--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 4608/40139[1G[################################------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 4968/40139[1G[##################################----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 5259/40139[1G[####################################--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 5625/40139[1G[#######################################-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 5978/40139[1G[#########################################---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 6272/40139[1G[###########################################-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 6632/40139[1G[#############################################-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 7013/40139[1G[###############################################---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 7321/40139[1G[##################################################------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 7673/40139[1G[####################################################----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 8033/40139[1G[######################################################--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 8331/40139[1G[########################################################------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 8704/40139[1G[###########################################################---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 9083/40139[1G[#############################################################-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 9457/40139[1G[################################################################----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 9829/40139[1G[##################################################################-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 10213/40139[1G[####################################################################-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 10597/40139[1G[#######################################################################--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 10981/40139[1G[#########################################################################------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 11365/40139[1G[############################################################################---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 11749/40139[1G[##############################################################################-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 12135/40139[1G[#################################################################################----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 12541/40139[1G[###################################################################################--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 12928/40139[1G[######################################################################################-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 13312/40139[1G[########################################################################################---------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 13696/40139[1G[###########################################################################################------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 14053/40139[1G[############################################################################################-----------------------------------------------------------------------------------------------------------------------------------------------------------------------] 14238/40139[1G[#############################################################################################----------------------------------------------------------------------------------------------------------------------------------------------------------------------] 14427/40139[1G[##############################################################################################---------------------------------------------------------------------------------------------------------------------------------------------------------------------] 14592/40139[1G[################################################################################################-------------------------------------------------------------------------------------------------------------------------------------------------------------------] 14845/40139[1G[#################################################################################################------------------------------------------------------------------------------------------------------------------------------------------------------------------] 15077/40139[1G[####################################################################################################---------------------------------------------------------------------------------------------------------------------------------------------------------------] 15479/40139[1G[#######################################################################################################------------------------------------------------------------------------------------------------------------------------------------------------------------] 15902/40139[1G[##########################################################################################################---------------------------------------------------------------------------------------------------------------------------------------------------------] 16357/40139[1G[############################################################################################################-------------------------------------------------------------------------------------------------------------------------------------------------------] 16768/40139[1G[###############################################################################################################----------------------------------------------------------------------------------------------------------------------------------------------------] 17200/40139[1G[##################################################################################################################-------------------------------------------------------------------------------------------------------------------------------------------------] 17637/40139[1G[####################################################################################################################-----------------------------------------------------------------------------------------------------------------------------------------------] 18048/40139[1G[#######################################################################################################################--------------------------------------------------------------------------------------------------------------------------------------------] 18500/40139[1G[##########################################################################################################################-----------------------------------------------------------------------------------------------------------------------------------------] 18934/40139[1G[#############################################################################################################################--------------------------------------------------------------------------------------------------------------------------------------] 19328/40139[1G[################################################################################################################################-----------------------------------------------------------------------------------------------------------------------------------] 19812/40139[1G[##################################################################################################################################---------------------------------------------------------------------------------------------------------------------------------] 20224/40139[1G[######################################################################################################################################-----------------------------------------------------------------------------------------------------------------------------] 20718/40139[1G[#########################################################################################################################################--------------------------------------------------------------------------------------------------------------------------] 21196/40139[1G[############################################################################################################################################-----------------------------------------------------------------------------------------------------------------------] 21670/40139[1G[###############################################################################################################################################--------------------------------------------------------------------------------------------------------------------] 22148/40139[1G[##################################################################################################################################################-----------------------------------------------------------------------------------------------------------------] 22650/40139[1G[#####################################################################################################################################################--------------------------------------------------------------------------------------------------------------] 23141/40139[1G[#########################################################################################################################################################----------------------------------------------------------------------------------------------------------] 23653/40139[1G[############################################################################################################################################################-------------------------------------------------------------------------------------------------------] 24165/40139[1G[###############################################################################################################################################################----------------------------------------------------------------------------------------------------] 24677/40139[1G[###################################################################################################################################################################------------------------------------------------------------------------------------------------] 25189/40139[1G[######################################################################################################################################################################---------------------------------------------------------------------------------------------] 25701/40139[1G[#########################################################################################################################################################################------------------------------------------------------------------------------------------] 26229/40139[1G[#############################################################################################################################################################################--------------------------------------------------------------------------------------] 26752/40139[1G[################################################################################################################################################################################-----------------------------------------------------------------------------------] 27294/40139[1G[####################################################################################################################################################################################-------------------------------------------------------------------------------] 27841/40139[1G[#######################################################################################################################################################################################----------------------------------------------------------------------------] 28385/40139[1G[##########################################################################################################################################################################################-------------------------------------------------------------------------] 28897/40139[1G[##############################################################################################################################################################################################---------------------------------------------------------------------] 29409/40139[1G[#################################################################################################################################################################################################------------------------------------------------------------------] 29840/40139[1G[####################################################################################################################################################################################################---------------------------------------------------------------] 30365/40139[1G[#######################################################################################################################################################################################################------------------------------------------------------------] 30828/40139[1G[##########################################################################################################################################################################################################---------------------------------------------------------] 31358/40139[1G[#############################################################################################################################################################################################################------------------------------------------------------] 31838/40139[1G[#################################################################################################################################################################################################################--------------------------------------------------] 32350/40139[1G[####################################################################################################################################################################################################################-----------------------------------------------] 32862/40139[1G[#######################################################################################################################################################################################################################--------------------------------------------] 33389/40139[1G[###########################################################################################################################################################################################################################----------------------------------------] 33949/40139[1G[###############################################################################################################################################################################################################################------------------------------------] 34526/40139[1G[##################################################################################################################################################################################################################################---------------------------------] 35056/40139[1G[#####################################################################################################################################################################################################################################------------------------------] 35550/40139[1G[#########################################################################################################################################################################################################################################--------------------------] 36062/40139[1G[#############################################################################################################################################################################################################################################----------------------] 36701/40139[1G[################################################################################################################################################################################################################################################-------------------] 37149/40139[1G[###################################################################################################################################################################################################################################################----------------] 37630/40139[1G[#######################################################################################################################################################################################################################################################------------] 38235/40139[1G[##########################################################################################################################################################################################################################################################---------] 38747/40139[1G[#############################################################################################################################################################################################################################################################------] 39260/40139[1G[#################################################################################################################################################################################################################################################################--] 39842/40139[2K[1G[2K[1G[1G[------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 0/1690[1G[##############################################################################################################################----------------------------------------------------------------------------------------------------------------------------------------] 812/1690[1G[##################################################################################################################################################################################################################---------------------------------------------------] 1358/1690[1G[#####################################################################################################################################################################################################################################################################] 1690/1690[2K[1G[2m[4/4][22m Building fresh packages...
[2K[1G
[2K[1G
[1G⠁ [0K[1G[1A⠁ [0K[1G[1B[1G[2A⠁ [0K[1G[2B[1G[2m[3/3][22m ⠂ now[0K[1G[1A[2m[2/3][22m ⠂ sharp[0K[1G[1B[1G[2A[2m[1/3][22m ⠂ core-js[0K[1G[2B[1G[2m[3/3][22m ⠄ now[0K[1G[1A[2m[2/3][22m ⠄ sharp[0K[1G[1B[1G[2A[2m[1/3][22m ⠄ core-js[0K[1G[2B[1G[2m[-/3][22m ⡀ waiting...[0K[1G[1A[2m[2/3][22m ⡀ sharp[0K[1G[1B[1G[2A[2m[1/3][22m ⡀ core-js[0K[1G[2B[1G[2m[-/3][22m ⢀ waiting...[0K[1G[1A[2m[2/3][22m ⢀ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⢀ waiting...[0K[1G[2B[1G[2m[-/3][22m ⠠ waiting...[0K[1G[1A[2m[2/3][22m ⠠ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⠠ waiting...[0K[1G[2B[1G[2m[-/3][22m ⠐ waiting...[0K[1G[1A[2m[2/3][22m ⠐ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⠐ waiting...[0K[1G[2B[1G[2m[-/3][22m ⠈ waiting...[0K[1G[1A[2m[2/3][22m ⠈ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⠈ waiting...[0K[1G[2B[1G[2m[-/3][22m ⠁ waiting...[0K[1G[1A[2m[2/3][22m ⠁ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⠁ waiting...[0K[1G[2B[1G[2m[-/3][22m ⠂ waiting...[0K[1G[1A[2m[2/3][22m ⠂ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⠂ waiting...[0K[1G[2B[1G[2m[-/3][22m ⠄ waiting...[0K[1G[1A[2m[2/3][22m ⠄ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⠄ waiting...[0K[1G[2B[1G[2m[-/3][22m ⡀ waiting...[0K[1G[1A[2m[2/3][22m ⡀ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⡀ waiting...[0K[1G[2B[1G[2m[-/3][22m ⢀ waiting...[0K[1G[1A[2m[2/3][22m ⢀ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⢀ waiting...[0K[1G[2B[1G[2m[-/3][22m ⠠ waiting...[0K[1G[1A[2m[2/3][22m ⠠ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⠠ waiting...[0K[1G[2B[1G[2m[-/3][22m ⠐ waiting...[0K[1G[1A[2m[2/3][22m ⠐ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⠐ waiting...[0K[1G[2B[1G[2m[-/3][22m ⠈ waiting...[0K[1G[1A[2m[2/3][22m ⠈ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⠈ waiting...[0K[1G[2B[1G[2m[-/3][22m ⠁ waiting...[0K[1G[1A[2m[2/3][22m ⠁ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⠁ waiting...[0K[1G[2B[1G[2m[-/3][22m ⠂ waiting...[0K[1G[1A[2m[2/3][22m ⠂ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⠂ waiting...[0K[1G[2B[1G[2m[-/3][22m ⠄ waiting...[0K[1G[1A[2m[2/3][22m ⠄ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⠄ waiting...[0K[1G[2B[1G[2m[-/3][22m ⡀ waiting...[0K[1G[1A[2m[2/3][22m ⡀ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⡀ waiting...[0K[1G[2B[1G[2m[-/3][22m ⢀ waiting...[0K[1G[1A[2m[2/3][22m ⢀ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⢀ waiting...[0K[1G[2B[1G[2m[-/3][22m ⠠ waiting...[0K[1G[1A[2m[2/3][22m ⠠ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⠠ waiting...[0K[1G[2B[1G[2m[-/3][22m ⠐ waiting...[0K[1G[1A[2m[2/3][22m ⠐ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⠐ waiting...[0K[1G[2B[1G[2m[-/3][22m ⠈ waiting...[0K[1G[1A[2m[2/3][22m ⠈ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⠈ waiting...[0K[1G[2B[1G[2m[-/3][22m ⠁ waiting...[0K[1G[1A[2m[2/3][22m ⠁ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⠁ waiting...[0K[1G[2B[1G[2m[-/3][22m ⠂ waiting...[0K[1G[1A[2m[2/3][22m ⠂ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⠂ waiting...[0K[1G[2B[1G[2m[-/3][22m ⠄ waiting...[0K[1G[1A[2m[2/3][22m ⠄ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⠄ waiting...[0K[1G[2B[1G[2m[-/3][22m ⡀ waiting...[0K[1G[1A[2m[2/3][22m ⡀ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⡀ waiting...[0K[1G[2B[1G[2m[-/3][22m ⢀ waiting...[0K[1G[1A[2m[2/3][22m ⢀ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⢀ waiting...[0K[1G[2B[1G[2m[-/3][22m ⠠ waiting...[0K[1G[1A[2m[2/3][22m ⠠ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⠠ waiting...[0K[1G[2B[1G[2m[-/3][22m ⠐ waiting...[0K[1G[1A[2m[2/3][22m ⠐ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⠐ waiting...[0K[1G[2B[1G[2m[-/3][22m ⠈ waiting...[0K[1G[1A[2m[2/3][22m ⠈ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⠈ waiting...[0K[1G[2B[1G[2m[-/3][22m ⠁ waiting...[0K[1G[1A[2m[2/3][22m ⠁ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⠁ waiting...[0K[1G[2B[1G[2m[-/3][22m ⠂ waiting...[0K[1G[1A[2m[2/3][22m ⠂ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⠂ waiting...[0K[1G[2B[1G[2m[-/3][22m ⠄ waiting...[0K[1G[1A[2m[2/3][22m ⠄ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⠄ waiting...[0K[1G[2B[1G[2m[-/3][22m ⡀ waiting...[0K[1G[1A[2m[2/3][22m ⡀ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⡀ waiting...[0K[1G[2B[1G[2m[-/3][22m ⢀ waiting...[0K[1G[1A[2m[2/3][22m ⢀ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⢀ waiting...[0K[1G[2B[1G[2m[-/3][22m ⠠ waiting...[0K[1G[1A[2m[2/3][22m ⠠ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⠠ waiting...[0K[1G[2B[1G[2m[-/3][22m ⠐ waiting...[0K[1G[1A[2m[2/3][22m ⠐ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⠐ waiting...[0K[1G[2B[1G[2m[-/3][22m ⠈ waiting...[0K[1G[1A[2m[2/3][22m ⠈ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⠈ waiting...[0K[1G[2B[1G[2m[-/3][22m ⠁ waiting...[0K[1G[1A[2m[2/3][22m ⠁ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⠁ waiting...[0K[1G[2B[1G[2m[-/3][22m ⠂ waiting...[0K[1G[1A[2m[2/3][22m ⠂ sharp[0K[1G[1B[1G[2A[2m[-/3][22m ⠂ waiting...[0K[1G[2B[1G[2m[-/3][22m ⠄ waiting...[0K[1G[1A[2m[2/3][22m ⠄ sharp[0K[1G[1B[1G[2A[2K[2B[1G[1A[2K[1B[2K[1G[2A[2K[1G[2K[1GDone in 12.62s.
]0;coder@jawnsy-m:~/projects/m/product/coder.com/site[?2004h[coder@jawnsy-m site]$ yarn build
[?2004l
[2K[1G[1myarn run v1.22.10[22m
[2K[1G[2m$ next build[22m
[33mwarn[39m  - React 17.0.1 or newer will be required to leverage all of the upcoming features in Next.js 11. Read more: https://err.sh/next.js/react-version
Loaded env from /home/coder/projects/m/product/coder.com/site/.env
[?25l[36minfo[39m  - Creating an optimized production build [36m.[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m..[39m[2K[1G[?25h[31mFailed to compile.[39m
[31m[39m
[?25l[36minfo[39m  - Creating an optimized production build [36m.[39m[2K[1G[?25h[36m./docs/scripts/add_clog_to_manifest.ts[39m:[33m7[39m:[33m23[39m
[31m[1mType error[22m[39m: Could not find a declaration file for module 'minimist'. '/home/coder/projects/m/product/coder.com/site/node_modules/minimist/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/minimist` if it exists or add a new declaration (.d.ts) file containing `declare module 'minimist';`

[0m [90m  5 | [39m[90m */[39m[0m
[0m [90m  6 | [39m[36mimport[39m { writeFileSync } from [32m"fs"[39m[33m;[39m[0m
[0m[31m[1m>[22m[39m[90m  7 | [39m[36mimport[39m parseArgs from [32m"minimist"[39m[33m;[39m[0m
[0m [90m    | [39m                      [31m[1m^[22m[39m[0m
[0m [90m  8 | [39m[36mimport[39m { resolve } from [32m"path"[39m[33m;[39m[0m
[0m [90m  9 | [39m[36mimport[39m semverCompare from [32m"semver-compare"[39m[33m;[39m[0m
[0m [90m 10 | [39m[36mimport[39m manifestJSON from [32m"../manifest.json"[39m[33m;[39m[0m
[?25l[36minfo[39m  - Creating an optimized production build [36m.[39m[?25h[2K[1G[31merror[39m Command failed with exit code 1.
[2K[1G[34minfo[39m Visit [1mhttps://yarnpkg.com/en/docs/cli/run[22m for documentation about this command.
]0;coder@jawnsy-m:~/projects/m/product/coder.com/site[?2004h[coder@jawnsy-m site]$ cd docs
[?2004l
]0;coder@jawnsy-m:~/projects/m/product/coder.com/site/docs[?2004h[coder@jawnsy-m docs]$ yarn
[?2004l
[2K[1G[1myarn install v1.22.10[22m
[2K[1G[33mwarning[39m package.json: No license field
[2K[1G[33mwarning[39m No license field
[2K[1G[2m[1/4][22m Resolving packages...
[1G⠁ [0K[2K[1G[2K[1G[2m[2/4][22m Fetching packages...
[2K[1G[1G[-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 0/181[1G[########################################################################################################-----------------------------------------------------------------------------] 104/181[2K[1G[2m[3/4][22m Linking dependencies...
[1G[#####################################################################################################################################################################################] 181/181[2K[1G[2K[1G[1G[-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 0/181[2K[1G[2K[1G[1G[-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 0/10432[1G[############---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 479/10432[1G[######################-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 896/10432[1G[###################################---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 1408/10432[1G[###############################################---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 1889/10432[1G[########################################################------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 2263/10432[1G[###################################################################-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 2688/10432[1G[###############################################################################-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 3189/10432[1G[###########################################################################################-------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 3639/10432[1G[###################################################################################################-----------------------------------------------------------------------------------------------------------------------------------------------------------------] 3968/10432[1G[############################################################################################################--------------------------------------------------------------------------------------------------------------------------------------------------------] 4352/10432[1G[######################################################################################################################----------------------------------------------------------------------------------------------------------------------------------------------] 4731/10432[1G[###############################################################################################################################-------------------------------------------------------------------------------------------------------------------------------------] 5109/10432[1G[############################################################################################################################################------------------------------------------------------------------------------------------------------------------------] 5627/10432[1G[##########################################################################################################################################################----------------------------------------------------------------------------------------------------------] 6193/10432[1G[#######################################################################################################################################################################---------------------------------------------------------------------------------------------] 6689/10432[1G[#####################################################################################################################################################################################-------------------------------------------------------------------------------] 7257/10432[1G[##################################################################################################################################################################################################------------------------------------------------------------------] 7767/10432[1G[############################################################################################################################################################################################################--------------------------------------------------------] 8187/10432[1G[#########################################################################################################################################################################################################################-------------------------------------------] 8695/10432[1G[######################################################################################################################################################################################################################################------------------------------] 9211/10432[1G[#####################################################################################################################################################################################################################################################---------------] 9823/10432[1G[#################################################################################################################################################################################################################################################################--] 10332/10432[2K[1G[2K[1G[1G[-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 0/352[2K[1G[2m[4/4][22m Building fresh packages...
[1B[1G[#######################################################################################################################################################################################################################################################################] 352/352[2K[1G[2K[1GDone in 2.36s.
]0;coder@jawnsy-m:~/projects/m/product/coder.com/site/docs[?2004h[coder@jawnsy-m docs]$ xs [K[K[Kcd ..
[?2004l
]0;coder@jawnsy-m:~/projects/m/product/coder.com/site[?2004h[coder@jawnsy-m site]$ yarn build
[?2004l
[2K[1G[1myarn run v1.22.10[22m
[2K[1G[2m$ next build[22m
[33mwarn[39m  - React 17.0.1 or newer will be required to leverage all of the upcoming features in Next.js 11. Read more: https://err.sh/next.js/react-version
Loaded env from /home/coder/projects/m/product/coder.com/site/.env
[?25l[36minfo[39m  - Creating an optimized production build [36m.[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m..[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m...[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m.[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m..[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m...[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m.[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m..[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m...[39m[2K[1G[?25h[36minfo[39m  - Using external babel configuration from /home/coder/projects/m/product/coder.com/site/.babelrc
[?25l[36minfo[39m  - Creating an optimized production build [36m.[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m..[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m...[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m.[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m..[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m...[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m.[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m..[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m...[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m.[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m..[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m...[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m.[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m..[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m...[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m.[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m..[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m...[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m.[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m..[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m...[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m.[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m..[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m...[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m.[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m..[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m...[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m.[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m..[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m...[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m.[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m..[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m...[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m.[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m..[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m...[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m.[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m..[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m...[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m.[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m..[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m...[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m.[39m[2K[1G[36minfo[39m  - Creating an optimized production build [36m..[39m[2K[1G[?25h[36minfo[39m  - Creating an optimized production build  
[33mwarn[39m  - Compiled with warnings

./src/utility/Docs.ts
Critical dependency: the request of a dependency is an expression

./docs/README.md
Error: Building... "/home/coder/projects/m/product/coder.com/site/docs/README.md". External assets are not allowed: https://img.shields.io/badge/all_contributors-20-orange.svg?style=flat-square

./docs/yarn.lock 1:0
Module parse failed: Unexpected character '#' (1:0)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
> # THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
| # yarn lockfile v1
|

./docs/scripts/create-clog.sh 1:0
Module parse failed: Unexpected character '#' (1:0)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
> #!/usr/bin/env bash
| #
| # create-clog.sh [--version=<version>] [--release-date=<release-date>]

[?25l[36minfo[39m  - Collecting page data [36m.[39m[2K[1G[?25h(node:766808) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
[?25l[36minfo[39m  - Collecting page data [36m.[39m[2K[1G[36minfo[39m  - Collecting page data [36m..[39m[2K[1G[36minfo[39m  - Collecting page data [36m...[39m[2K[1G[36minfo[39m  - Collecting page data [36m.[39m[2K[1G[36minfo[39m  - Collecting page data [36m..[39m[2K[1G[36minfo[39m  - Collecting page data [36m...[39m[2K[1G[36minfo[39m  - Collecting page data [36m.[39m[2K[1G[36minfo[39m  - Collecting page data [36m..[39m[2K[1G[36minfo[39m  - Collecting page data [36m...[39m(node:768217) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:768266) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:768182) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:768280) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:768287) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:768273) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:768315) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:768196) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:768339) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:768189) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:768294) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:768353) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:768301) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:768346) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:768308) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:768210) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:768418) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:768432) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:768203) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:768425) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:768332) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:768386) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:768404) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:768411) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:768234) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:768446) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:768439) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
[2K[1G[36minfo[39m  - Collecting page data [36m.[39m(node:768453) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
[2K[1G[36minfo[39m  - Collecting page data [36m..[39mresponse from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
[2K[1G[?25h[36minfo[39m  - Collecting page data  
[?25l[36m[    ][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[=   ][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[==  ][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[=== ][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[ ===][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[  ==][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[   =][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[    ][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[   =][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[  ==][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[ ===][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[====][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[=== ][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[==  ][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[=   ][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[    ][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[=   ][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[==  ][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[=== ][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[ ===][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[  ==][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[   =][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[    ][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[   =][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[  ==][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[ ===][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[====][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[=== ][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[==  ][39m [36minfo[39m  - Generating static pages (0/203)(node:766808) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:766808) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
[2K[1G[36m[=   ][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[    ][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[=   ][39m [36minfo[39m  - Generating static pages (0/203)(node:766808) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:766808) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
[2K[1G[36m[==  ][39m [36minfo[39m  - Generating static pages (0/203)(node:766808) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:766808) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
[2K[1G[36m[=== ][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[ ===][39m [36minfo[39m  - Generating static pages (0/203)response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
(node:766808) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:766808) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:766808) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
[2K[1G[36m[  ==][39m [36minfo[39m  - Generating static pages (0/203)(node:766808) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
[2K[1G[36m[   =][39m [36minfo[39m  - Generating static pages (0/203)(node:766808) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:766808) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
[2K[1G[36m[    ][39m [36minfo[39m  - Generating static pages (0/203)(node:766808) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
[2K[1G[36m[   =][39m [36minfo[39m  - Generating static pages (0/203)(node:766808) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
[2K[1G[36m[  ==][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[ ===][39m [36minfo[39m  - Generating static pages (0/203)(node:766808) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
[2K[1G[36m[====][39m [36minfo[39m  - Generating static pages (0/203)(node:766808) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:766808) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
[2K[1G[36m[=== ][39m [36minfo[39m  - Generating static pages (0/203)response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
[2K[1G[36m[==  ][39m [36minfo[39m  - Generating static pages (0/203)response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
(node:766808) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
[2K[1G[36m[=   ][39m [36minfo[39m  - Generating static pages (0/203)(node:766808) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:766808) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
[2K[1G[36m[    ][39m [36minfo[39m  - Generating static pages (0/203)response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
[2K[1G[36m[=   ][39m [36minfo[39m  - Generating static pages (0/203)response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
[2K[1G[36m[==  ][39m [36minfo[39m  - Generating static pages (0/203)response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
[2K[1G[36m[=== ][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[ ===][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[  ==][39m [36minfo[39m  - Generating static pages (0/203)(node:766808) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:766808) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:766808) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
[2K[1G[36m[   =][39m [36minfo[39m  - Generating static pages (0/203)(node:766808) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:766808) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:766808) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
[2K[1G[36m[    ][39m [36minfo[39m  - Generating static pages (0/203)(node:766808) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
[2K[1G[36m[   =][39m [36minfo[39m  - Generating static pages (0/203)(node:766808) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
(node:766808) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
[2K[1G[36m[  ==][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[ ===][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[====][39m [36minfo[39m  - Generating static pages (0/203)[2K[1G[36m[=== ][39m [36minfo[39m  - Generating static pages (50/203)(node:766808) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
[2K[1G[36m[==  ][39m [36minfo[39m  - Generating static pages (50/203)(node:766808) ExperimentalWarning: Package name self resolution is an experimental feature. This feature could change at any time
[2K[1G[36m[=   ][39m [36minfo[39m  - Generating static pages (50/203)[2K[1G[36m[    ][39m [36minfo[39m  - Generating static pages (50/203)[2K[1G[36m[=   ][39m [36minfo[39m  - Generating static pages (50/203)[2K[1G[36m[==  ][39m [36minfo[39m  - Generating static pages (50/203)[2K[1G[36m[=== ][39m [36minfo[39m  - Generating static pages (50/203)[2K[1G[36m[ ===][39m [36minfo[39m  - Generating static pages (50/203)[2K[1G[36m[  ==][39m [36minfo[39m  - Generating static pages (50/203)[2K[1G[36m[   =][39m [36minfo[39m  - Generating static pages (101/203)[2K[1G[36m[    ][39m [36minfo[39m  - Generating static pages (101/203)response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
[2K[1G[36m[   =][39m [36minfo[39m  - Generating static pages (101/203)response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
[2K[1G[36m[  ==][39m [36minfo[39m  - Generating static pages (101/203)response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
[2K[1G[36m[ ===][39m [36minfo[39m  - Generating static pages (101/203)response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
[2K[1G[36m[====][39m [36minfo[39m  - Generating static pages (101/203)response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
[2K[1G[36m[=== ][39m [36minfo[39m  - Generating static pages (101/203)response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
[2K[1G[36m[==  ][39m [36minfo[39m  - Generating static pages (101/203)response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
[2K[1G[36m[=   ][39m [36minfo[39m  - Generating static pages (101/203)response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
[2K[1G[36m[    ][39m [36minfo[39m  - Generating static pages (101/203)response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
[2K[1G[36m[=   ][39m [36minfo[39m  - Generating static pages (101/203)[2K[1G[36m[==  ][39m [36minfo[39m  - Generating static pages (152/203)response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
[2K[1G[36m[=== ][39m [36minfo[39m  - Generating static pages (152/203)response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
[2K[1G[36m[ ===][39m [36minfo[39m  - Generating static pages (152/203)response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
[2K[1G[36m[  ==][39m [36minfo[39m  - Generating static pages (152/203)response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
[2K[1G[36m[   =][39m [36minfo[39m  - Generating static pages (152/203)response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
[2K[1G[36m[    ][39m [36minfo[39m  - Generating static pages (152/203)response from CMS for https://graphql.datocms.com/environments/master/preview : 200 OK
[2K[1G[36m[   =][39m [36minfo[39m  - Generating static pages (152/203)[2K[1G[?25h[36minfo[39m  - Generating static pages (203/203)
[?25l[36minfo[39m  - Finalizing page optimization [36m.[39m[2K[1G[36minfo[39m  - Finalizing page optimization [36m..[39m[2K[1G[?25h[36minfo[39m  - Finalizing page optimization  

[4mPage[24m                                                                           [4mSize[24m     [4mFirst Load JS[24m
┌ ● /                                                                          8.63 kB         [31m[1m201 kB[22m[39m
├   /_app                                                                      0 B             [33m149 kB[39m
├ ○ /_enterprise                                                               3.66 kB         [31m[1m189 kB[22m[39m
├ ○ /404                                                                       5.66 kB         [33m154 kB[39m
├ λ /api/jobs                                                                  0 B             [33m149 kB[39m
├ λ /api/license                                                               0 B             [33m149 kB[39m
├ λ /api/metrics                                                               0 B             [33m149 kB[39m
├ λ /api/posts/[cat]/[pid]                                                     0 B             [33m149 kB[39m
├ λ /api/posts/author/[...params]                                              0 B             [33m149 kB[39m
├ λ /api/preview                                                               0 B             [33m149 kB[39m
├ λ /api/repoStats                                                             0 B             [33m149 kB[39m
├ λ /api/validateEmail                                                         0 B             [33m149 kB[39m
├ ● /blog                                                                      5.07 kB         [31m[1m277 kB[22m[39m
├ ● /blog/[slug]                                                               5.38 kB         [31m[1m282 kB[22m[39m
├   ├ /blog/coder-welcomes-ketan-gangatirkar-as-vice-president-of-engineering
├   ├ /blog/coder-v1-17
├   ├ /blog/create-cloud-dev-environments-in-5-minutes-with-coder
├   └ [+41 more paths]
├ ● /blog/author/[alias]                                                       1.52 kB         [31m[1m278 kB[22m[39m
├   ├ /blog/author/team
├   ├ /blog/author/erik
├   ├ /blog/author/thomas
├   └ [+13 more paths]
├ ○ /company                                                                   12 kB           [31m[1m198 kB[22m[39m
├ ● /contact                                                                   4.68 kB         [31m[1m193 kB[22m[39m
├ ● /demo                                                                      4.64 kB         [31m[1m211 kB[22m[39m
├ ● /demo/thanks                                                               659 B           [31m[1m189 kB[22m[39m
├ ● /devsecops                                                                 6.87 kB         [31m[1m199 kB[22m[39m
├ ● /docs/[[...slug]]                                                          63.4 kB         [31m[1m252 kB[22m[39m
├   └ css/9504b1791144bf0d6adb.css                                             6.1 kB
├   ├ /docs/about
├   ├ /docs/comparison
├   ├ /docs/feedback
├   └ [+101 more paths]
├ ○ /legal/open-source-licenses                                                26.5 kB         [31m[1m212 kB[22m[39m
├ ○ /legal/privacy-policy                                                      7.71 kB         [31m[1m194 kB[22m[39m
├ ○ /legal/terms-of-service                                                    7.17 kB         [31m[1m193 kB[22m[39m
├ ● /pricing                                                                   7.48 kB         [31m[1m228 kB[22m[39m
├ ● /pricing/thanks                                                            691 B           [31m[1m189 kB[22m[39m
├ ● /public-sector                                                             10.7 kB         [31m[1m203 kB[22m[39m
├ ● /resources/videos                                                          3.04 kB         [31m[1m274 kB[22m[39m
├ ● /resources/webinars                                                        3.04 kB         [31m[1m274 kB[22m[39m
├ ● /resources/whitepapers                                                     3.04 kB         [31m[1m274 kB[22m[39m
├ λ /sitemap.xml                                                               2.8 kB          [33m151 kB[39m
├ ● /trial                                                                     167 B           [31m[1m207 kB[22m[39m
├ ● /trial/end                                                                 1.3 kB          [31m[1m190 kB[22m[39m
├ ● /trial/start                                                               9.19 kB         [31m[1m230 kB[22m[39m
├ ● /trial/thanks                                                              1.16 kB         [31m[1m190 kB[22m[39m
├ ● /videos/[slug]                                                             2.97 kB         [31m[1m274 kB[22m[39m
├   ├ /videos/build-custom-images-for-coder
├   ├ /videos/local-editors-with-coder
├   ├ /videos/coffee-and-coder-march-2021
├   └ [+8 more paths]
├ ● /webinars/[slug]                                                           34.9 kB         [31m[1m320 kB[22m[39m
├   └ /webinars/the-coder-secure-by-default-developer-experience
├ ● /webinars/[slug]/thanks                                                    2.1 kB          [31m[1m191 kB[22m[39m
├   └ /webinars/the-coder-secure-by-default-developer-experience/thanks
└ ● /whitepapers/[slug]                                                        3.73 kB         [31m[1m206 kB[22m[39m
    ├ /whitepapers/software-development-in-the-cloud-how-to-get-there
    ├ /whitepapers/devsecops-playbook
    ├ /whitepapers/zero-trust-security-for-software-development-teams
    └ /whitepapers/ciso-whitepaper-security-productivity
+ First Load JS shared by all                                                  [33m149 kB[39m
  ├ chunks/aee0a6ff891c81e1fbc8e81ce2de407a64c80e38.cc123e.js                  80.3 kB
  ├ chunks/bd62a74edba2480146559a4ae64f057fad336f79.3192d0.js                  11.8 kB
  ├ chunks/framework.73e439.js                                                 40.2 kB
  ├ chunks/main.dbbac5.js                                                      5.81 kB
  ├ chunks/pages/_app.bb4ebd.js                                                9.72 kB
  ├ chunks/webpack.50bee0.js                                                   751 B
  └ css/be932e8a9249f782d0eb.css                                               169 B

λ  (Server)  server-side renders at runtime (uses [36mgetInitialProps[39m or [36mgetServerSideProps[39m)
○  (Static)  automatically rendered as static HTML (uses no initial props)
●  (SSG)     automatically generated as static HTML + JSON (uses [36mgetStaticProps[39m)
   (ISR)     incremental static regeneration (uses revalidate in [36mgetStaticProps[39m)

[?25h[2K[1GDone in 52.61s.
]0;coder@jawnsy-m:~/projects/m/product/coder.com/site[?2004h[coder@jawnsy-m site]$ exit
[?2004l
exit

Script done on 2021-03-29 20:12:49+00:00 [COMMAND_EXIT_CODE="0"]
```

Reply:
Having to add dependencies in `m/product/coder.com/site` to be used here does not seem like a good course of action. Dependencies for this module should be definable within this module, and not the outside world.

> or modify that file to also install the dependencies in `docs` when the top-level build runs.

We do this when an install is run in the root of `m/`. I suppose we can do it in `product/coder.com/site` as well.

Reply:
> Dependencies for this module should be definable within this module, and not the outside world.

I agree with that, yeah...

I didn't know about the post-install script magic until you pointed it out. I think Vercel is rebuilding everything from our source code, so I didn't think caching would play a part here?

Reply:
let's move this thread to #208 as I think it was discovered this dupe PR did not help in any way, and re-opening a PR seems to trigger a new run of CI. That was my mistake for creating dupes - I hadn't realized how convenient it was to open/re-open a single PR.

I think before we merge #208 (i can get by - I'll just checkout the changes locally but not push them to release branch tomorrow), we should really take time and go through some of these CI and build considerations.

The fact that `coder.com/site` is parsing a ts file in `docs` is weird to me.